proto/ibc/apps/cross-chain-validation/tx.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
```

Comment:
All of these messages will likely be implemented and processed in the new staking module, but I've defined them here for now.

Reply:
redelegation is missing

Reply:
redelegation in what sense? If you mean redelegation as in a delegator moving their stake from one validator to another, this is not really in purview of CCV. The staking keeper will keep track of this and propogate any valset changes to CCV.

If you mean a validator wants to move some of their stake away from babyChainA and stake those atoms on babyChainB, I think we could support that without waiting for the unbondingperiod to be over since the atoms will still be staked. Need to verify with spec team.